composer.json

```diff
@@ -14,7 +14,8 @@
 	"require": {
 		"php": "^8.0 || ^7.3",
 		"guzzlehttp/guzzle": "^6.0 || ^7.0",
-		"ext-json": "*"
+		"ext-json": "*",
+		"psr/http-client": "^1.0"

```

Comment:
Changed to correct package now.

How can we let guzzle determine the version? Right now, we have the exact same version query as guzzle (^1.0).

Reply:
I can use a string, as you did before, instead of the `::class` constant, and remove the package requirement, if you want.

I just, in general, personally like code better than strings, but it will do no big difference in this case.

Reply:
> I can use a string, as you did before, instead of the `::class` constant, and remove the package requirement, if you want.
> 
> I just, in general, personally like code better than strings, but it will do no big difference in this case.

I also prefer the `::class` constant. We'll be removing the string in v2.